source/views/dictionary/list.js

```diff
@@ -130,6 +130,7 @@ export class DictionaryView extends React.Component {
         <SearchBar
           getRef={ref => (this.searchBar = ref)}
           onChangeText={this.performSearch}
+          onSearchButtonPress={() => this.searchBar.unFocus()}
```

Comment:
actually one request – could you change this to just `{this.searchBar.unFocus}`, please? Passing in the function reference instead of creating an anonymous function just to call it?